src/style.css

```diff
@@ -0,0 +1,109 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  list-style: none;
+}
+
+html {
+  background-color: #f5f1f1;
+}
+
+.section_todo {
+  margin: 50px 10% 0 10%;
+  background-color: #fff;
```

Comment:
@HFG43 Hernan, since your `section_todo` selector and your `#reload` use the same `background-color` property, you can chain both selectors to use it and avoid style repetition 😃.
e.g.
```css
.section_todo,
#reload {
    background-color: #fff;
}
```
The same could apply to the `border-bottom` property implemented in some selectors.

Happy coding!

Reply:
@ITurres Thanks Arthur! I have followed your advice to avoid repetition and created both new classess!